src/util/resize.js

```diff
@@ -0,0 +1,50 @@
+export const DIVIDER_HEIGHT = 4;
+export const DEFAULT_VERTICAL_FLEX = [
+  `1 1 ${DIVIDER_HEIGHT}px`,
+  `1 1 ${DIVIDER_HEIGHT}px`,
```

Comment:
Yup your third comment has it. I think a heavily simplified version should make what's happening clearer:
```jsx
<EditorsColumn>
  <EditorContainer flex='?'>
    <Editor />
    <Divider height='4px'/>
  </EditorContainer>
  <EditorContainer flex='?'>
    <Editor />
    <Divider height='4px'/>
  </EditorContainer>
  <EditorContainer flex='?'>
    <Editor />
  </EditorContainer>
</EditorsColumn>
```
Our `flex` is applied to `EditorContainer`; if we use a default of `1`, (or the equivalent `1 1 0` or the `0 1 100vh` you mention) then each `EditorContainer` gets the same height.

The trouble is that that's not our goal; our goal is to give each *`Editor`* the same height. 

If we give each `EditorContainer` the same height, then the 3rd one will give its `Editor` the full height. However, the 1st `EditorContainer` has to fit both an `Editor` *and* a `Divider` in its allotted height. The `Divider` height is inflexible, so its the height of `Editor` that gets cut by the height of `Divider`. The same goes for the 2nd container.

As a concrete example, if we have an `EditorsColumn` of height `900px`, then each `EditorContainer` will get height `300px`, but the top two `Editor`s will only get height `296px` while the 3rd will be `300px`.

The solution to this lies in altering the `flex-basis` of the three `EditorContainer`s. By giving the 1st and 2nd containers a boosted starting point (boosted by the height of the `Divider`), they'll take up a correspondingly larger allottment of the available height. This way, given an available height of `908px`, the first two `EditorContainer`s will get height `304px` while the 3rd gets height `300px`, and all `Editor`s get height `300px`.

I hope that helps, and that I'm not missing something blindingly obvious...